server/server.go

```diff
@@ -1048,14 +1048,6 @@ func (s *Server) setSystemAccount(acc *Account) error {
 	if acc.imports.services == nil {
 		acc.imports.services = make(map[string]*serviceImport)
 	}
-
-	// Create a dummy internal client for fast lookup for
```

Comment:
Since I have removed randomClient() (not needed anymore), I removed this because you added with this comment. However, if this is still required (regardless of randomClient()), I can put that back.
(I notice that there is quite a bit of flapping, so not sure if this is due to that code being removed).

Reply:
I believe I had just added that with last PR, so should be fine to remove.